### PR TITLE
Feature null reaction dir

### PR DESCRIPTION
**Pull request by EnnaDelfen**
This branch reinstate the "Null" reaction folder so that when "no chemistry" is needed (in the sense that no reactions occur, not necessarily mean we have no species) then react() simply does nothing.
Makes it possible to compile non reacting test cases without sundials.
Does not break anything or changes any structure in place currently in any of the Pele codes

### CODE REVIEW

File: Testing/Exec/Make.PelePhysics [hunk 1]
```diff
@@ -71,7 +71,7 @@ ifeq ($(USE_F90_PP),TRUE)
     INCLUDE_LOCATIONS += $(REACTIONS_HOME_F90) $(REACTIONS_PATH_F90)
     VPATH_LOCATIONS   += $(REACTIONS_HOME_F90) $(REACTIONS_PATH_F90)
 endif
-ifdef Chemistry_Model
+#ifdef Chemistry_Model
```
Comment (drummerdoc) [inline Testing/Exec/Make.PelePhysics]: Seems weird to have to use defines in GNUmakefiles

**Review by drummerdoc — APPROVED** [review]

File: Testing/Exec/Make.PelePhysics (hunk 1 above)
Comment (EnnaDelfen) [reply Testing/Exec/Make.PelePhysics]: Yes, I commented them out in case but I'll remove them completely.

**Comment by EnnaDelfen** [comment]
Waiting on Lucas to test it on a PLM case

**Review by esclapez — APPROVED** [review]
Looks fine to me. Gives the same results as going through hack_nochem in PeleLM (but HR is not hard zero because of machine precision operations).

**Pull request merged**